broker-cli/commands/wallet.js

```diff
@@ -300,16 +300,37 @@ function formatBalance (balance, status) {
  * @return {Void}
  */
 async function release (args, opts, logger) {
-  const { rpcAddress, market } = opts
+  const { market } = args
+  const { rpcAddress, force } = opts
 
   try {
     const client = new BrokerDaemonClient(rpcAddress)
 
-    const answer = await askQuestion(`Are you sure you want to close all channels you have open on the ${market} market? (Y/N)`)
+    let question
+    if (force) {
+      question = `Are you sure you want to FORCE the release of all channels you have open on the ${market} market? (Y/N)`
+    } else {
+      question = `Are you sure you want to release all channels you have open on the ${market} market? (Y/N)`
+    }
+
+    const answer = await askQuestion(question)
 
     if (!ACCEPTED_ANSWERS.includes(answer.toLowerCase())) return
 
-    await client.walletService.releaseChannels({market})
+    try {
+      await client.walletService.releaseChannels({ market, force })
+    } catch (e) {
+      logger.error(`Failed to release payment channels for ${market}`.red)
```

Comment:
Do we want to show what the error is before we advise them to force close?

Reply:
👍

Reply:
@treygriffith @martineehrlich the error gets rethrown after the text.

Reply:
Yes, but I was suggesting that we should show/log what the error is here, and still throw after.